lib/msf/core/payload/uuid/options.rb

```diff
@@ -47,12 +47,10 @@ def generate_uri_uuid_mode(mode, len = nil, uuid: nil)
   end
 
   # Generate a Payload UUID
-  def generate_payload_uuid
+  def generate_payload_uuid(conf = {})
```

Comment:
It's necessary for the new payloads to explicitly set the architecture and platform. The routine will take the architecture and platform array and only encode the first entry. This results in the session being identified as `cmd/unix`, and results in failures because stdapi can't load.